src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -1551,7 +1551,7 @@ public virtual VirtualFileResult File(string virtualPath, string contentType, st
         /// This supports range requests (<see cref="StatusCodes.Status206PartialContent"/> or
         /// <see cref="StatusCodes.Status416RangeNotSatisfiable"/> if the range is not satisfiable).
         /// </summary>
-        /// <param name="physicalPath">The physical path of the file to be returned.</param>
+        /// <param name="physicalPath">The path to the file. The path must be an absolute path.</param>
```

Comment:
Took the comment from `PhyiscalFileResult` as its more clearer than what we already have.